pkg/pillar/containerd/containerd.go

```diff
@@ -557,6 +557,10 @@ func CtrDelete(containerID string) error {
 	// do this just in case
 	_ = CtrStop(containerID, true)
 
+	stdinDir := path.Join("/run", "containers-stdin", containerID)
+	if err := os.RemoveAll(stdinDir); err != nil {
```

Comment:
Why are you doing this?

Reply:
Just cleaning it up, since the next CtrCreate expects these directories to be not present. This got exposed when LKTaskLaunch() deletes existing container (if any) before creating new container with the same domainID

Reply:
I think we need to dig into this more @cshari-zededa -- since it may expose an issue with how we're using containerd and I'd rather fix the root cause (since containerd becomes a center piece of our design). Is there an easy way to reproduce this?

Reply:
It is not reproducible, I think I added this change unnecessarily. Removed it!

Reply:
Hi @rvs, 
With the [#containerd issue ](https://github.com/containerd/containerd/issues/4019) fixed, 
we no longer need this stdin workaround, so removed that piece of code, and this cleanup 
code as well. Please let me know if you think we should retain it.